app/controllers/sites_controller.rb

```diff
@@ -27,6 +28,16 @@ def create
     end
   end
 
+  def update
+    @site = Site.find(params[:id])
+    if @site.update(site_params)
+      redirect_to site_path(@site)
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.